extensions/sql-migration/src/wizard/wizardController.ts

```diff
@@ -79,7 +79,9 @@ export class WizardController {
 		});
 
 		await Promise.all(wizardSetupPromises);
-		await pages[0].onPageEnter();
+		wizard.onPageChanged(async (pageChangeInfo: azdata.window.WizardPageChangeInfo) => {
+			await pages[0].onPageEnter(pageChangeInfo);
```

Comment:
did this get merged with latest, I was expecting to see this:
this.extensionContext.subscriptions.push(wizard.onPageChanged(async (pageChangeInfo: azdata.window.WizardPageChangeInfo) => {

Reply:
merged with the latest and resolved this issue